src/models/graph.ts

```diff
@@ -274,7 +273,35 @@ export class Graph<N extends INodeBase, E extends IEdgeBase> implements IGraph<N
     this._applyEdgeOffsets();
     this._applyStyle();
 
-    this._settings?.onRemoveData?.(data);
+    if (this._settings && this._settings.onRemoveData) {
+      this._settings.onRemoveData(data);
```

Comment:
As a user (internal or external) I would expect to receive in the callback all nodes and edges that have been removed. You are forwarding the data that you've received which doesn't necessarily mean that those have been removed, e.g.

In my graph I have these nodes and edges
- Nodes: 1, 2, 3
- Edges A (1-2), B (2-1), C(1-3)

I call remove with `nodes = 1, 3, 5` and `edges = C, D`. But actually nodes `1, 3` will be removed and all edges will be removed (`A, B, C`). So instead of receiving `[1, 3, 5], [C, D]` in the callback, I would expect to receive `[1, 3], [A, B, C]`.